### PR TITLE
git-standup 1.0.1

### PR DESCRIPTION
**Pull request by nicosomb**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
### Description

Update git-standup 1.0.1

### CODE REVIEW

File: Formula/git-standup.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class GitStandup < Formula
   desc "Git extension to generate reports for standup meetings"
   homepage "https://github.com/kamranahmedse/git-standup"
-  url "https://github.com/kamranahmedse/git-standup/archive/1.0.0.tar.gz"
-  sha256 "fd3808fb8414308413248b2f47646f51ce7bf7a6b59beb257fbde2e983b60127"
+  url "https://github.com/kamranahmedse/git-standup/archive/1.0.1.tar.gz"
+  sha256 "0c43e7414b308bc29ef6f04e84b02aaf3e0ac4837522d34f84359a54c0a4d54b"
   head "https://github.com/kamranahmedse/git-standup.git"
 
   bottle :unneeded
```
Comment (mikemcquaid) [inline Formula/git-standup.rb]: Remove this line and use `system "make", "install", "prefix=#{prefix}"` below.

Comment (nicosomb) [inline Formula/git-standup.rb]: You mean

```
class GitStandup < Formula
  desc "Git extension to generate reports for standup meetings"
  homepage "https://github.com/kamranahmedse/git-standup"
  url "https://github.com/kamranahmedse/git-standup/archive/1.0.1.tar.gz"
  sha256 "0c43e7414b308bc29ef6f04e84b02aaf3e0ac4837522d34f84359a54c0a4d54b"
  head "https://github.com/kamranahmedse/git-standup.git"

  system "make", "install", "prefix=#{prefix}"

  def install
    bin.install "git-standup"
  end

  test do
    system "git", "init"
    (testpath/"test").write "test"
    system "git", "add", "#{testpath}/test"
    system "git", "commit", "--message", "test"
    system "git", "standup", "HEAD"
  end
end
```

?

Comment (mikemcquaid) [inline Formula/git-standup.rb]: No, sorry, use that `system` line instead of `bin.install`

Comment (nicosomb) [inline Formula/git-standup.rb]: I prefer to be sure before committing.
You mean: 

```
class GitStandup < Formula
  desc "Git extension to generate reports for standup meetings"
  homepage "https://github.com/kamranahmedse/git-standup"
  url "https://github.com/kamranahmedse/git-standup/archive/1.0.1.tar.gz"
  sha256 "0c43e7414b308bc29ef6f04e84b02aaf3e0ac4837522d34f84359a54c0a4d54b"
  head "https://github.com/kamranahmedse/git-standup.git"

  bottle :unneeded

  def install
    system "make", "install", "prefix=#{prefix}"
  end

  test do
    system "git", "init"
    (testpath/"test").write "test"
    system "git", "add", "#{testpath}/test"
    system "git", "commit", "--message", "test"
    system "git", "standup", "HEAD"
  end
end
```

Comment (mikemcquaid) [inline Formula/git-standup.rb]: The `install` is now correct but `bottle :unneeded` should be removed.

Comment (nicosomb) [inline Formula/git-standup.rb]: Done.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**